examples/keygen/keyimport.c

```diff
@@ -55,6 +86,8 @@ int TPM2_Keyimport_Example(void* userCtx, int argc, char *argv[])
     TPM_ALG_ID paramEncAlg = TPM_ALG_NULL;
     WOLFTPM2_SESSION tpmSession;
     const char* outputFile = "keyblob.bin";
+    uint8_t derEncode = 0;
```

Comment:
I like stdint.h types, but please use `byte`. Thanks

Reply:
so is it only wolfBoot that uses uint types?